### PR TITLE
Clarify Enterprise required for multiple instances

### PR DESCRIPTION
**Pull request by bbodenmiller**

### CODE REVIEW

**Review by jfrerich — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This plugin supports a two-way integration between Mattermost and Jira. Jira Core and Jira Software products, for Server, Data Center, and Cloud platforms are supported. It has been tested with versions 7 and 8.
 
-From v3.0 of this plugin, support for multiple Jira instances is offered.
+From v3.0 of this plugin, support for multiple Jira instances is offered when using an Enterprise License.
```
Comment (justinegeffen) [inline readme.md:5]:
```suggestion
From v3.0 of this plugin, support for multiple Jira instances is offered for Mattermost Server Enterprise Edition and Mattermost Cloud users.
```

Comment (justinegeffen) [reply readme.md:5]: @jfrerich, is this correct?

Comment (jfrerich) [reply readme.md:5]: I think you could leave it a `for Mattermost Enterprise Editions`

Comment (bbodenmiller) [reply readme.md:5]:
```suggestion
From v3.0 of this plugin, support for multiple Jira instances is offered for Mattermost Enterprise Editions.
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**